debian/debathena-pyhesiodfs.service

```diff
@@ -0,0 +1,20 @@
+# Author: Lizhou Sha <slz@mit.edu>
+# This file is part of the debathena-pyhesiodfs package.
+
+[Unit]
+Description=Hesiod automounter for Athena lockers
+Requires=openafs-client.service
```

Comment:
Also, do I `Wants=network-online.target` here, following `openafs-client.service`? `man systemd.special` seems to suggest that remote file systems should pull that in, and also use `After=network-online.target`.